aws/cloudformation/scylla.yaml.j2

```diff
@@ -177,33 +191,29 @@ Resources:
               --totalnodes 1 --reflector example.com
 {%- endif %}
       NetworkInterfaces:
-        - NetworkInterfaceId: !Ref Node{{ node_index }}ETH0
+        - AssociatePublicIpAddress: !If [EnablePublicAccess, true, false]
```

Comment:
I think it can be shortened by using the parameter value directly `!Ref EnablePublicAccess` and remove the `EnablePublicAccess` condition on line 88